equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/service/CrmFileService.java

```diff
@@ -42,59 +42,41 @@ private static Class<? extends CrmFormModelInterface> getCrmFormTypeMapClass(Int
         };
     }
 
-    public <T extends CrmFormModelInterface> T getCrmImageFile(CrmFormDetailsCriteriaDTO crmFormDetailsCriteriaDTO) {
+    public <T extends CrmFormModelInterface> T getCrmFormData(CrmFormDetailsCriteriaDTO crmFormDetailsCriteriaDTO) {
         JSONObject crmFileJsonObject = getCrmFileJson(crmFormDetailsCriteriaDTO);
 
-        return convertCrmFormJsonToModel(crmFileJsonObject, crmFormDetailsCriteriaDTO);
+        // Format sanity checks and conversions
+        crmFileJsonObject.put("linkedAttachments", convertCrmFormLinkedAttachments(crmFileJsonObject));
+
+        return convertCrmFileJsonToModel(crmFileJsonObject, crmFormDetailsCriteriaDTO);
     }
 
-    private JSONObject getCrmFileJson(CrmFormDetailsCriteriaDTO crmFormDetailsCriteriaDTO) throws JSONException {
+    public JSONObject getCrmFileJson(CrmFormDetailsCriteriaDTO crmFormDetailsCriteriaDTO) throws JSONException {
         TaskImageFilesModel task = (TaskImageFilesModel) taskImageFilesRepository.findOne(
-                    crmFormDetailsCriteria.getSpecification(crmFormDetailsCriteriaDTO)
+                crmFormDetailsCriteria.getSpecification(crmFormDetailsCriteriaDTO)
             )
             .orElseThrow(() -> new ResourceNotFoundException("Task with USN " + crmFormDetailsCriteriaDTO.usn() + " not found"));
 
-
         // Collect and clean content
-        String odfImageContentString = new String(task.getCrmFile(), StandardCharsets.UTF_8)
-                .replaceAll("\u0000", "");
+        String crmFormFileContent = new String(task.getCrmFile(), StandardCharsets.UTF_8)
+            .replaceAll("\u0000", "");
```

Comment:
Can you put `"\u0000"`  in a string variable - private static final string  ?

Reply:
done